nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -478,7 +478,7 @@ public void compareGenericTypeParameterNullabilityForCall(
       // all remaining actual arguments in the next loop.
       n = n - 1;
     }
-    for (int i = 0; i < n - 1; i++) {
+    for (int i = 0; i < n; i++) {
```

Comment:
Ah, I see, `n` has already been decremented for the case where the last formal is varargs, so this should always be looping up to `n`. Right?

Reply:
Yes that's right